Dockerfile

```diff
@@ -30,3 +30,22 @@ ENV JAVA_HOME /usr/lib/jvm/default-java
 # Set up baselisk.
 RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v1.10.1/bazelisk-linux-amd64 \
     -O /usr/local/bin/bazelisk && chmod a+x /usr/local/bin/bazelisk
+
+# Rust is needed for authorization logic
+ARG rustup_dir=/usr/local/cargo
+ENV RUSTUP_HOME ${rustup_dir}
+ENV CARGO_HOME ${rustup_dir}
+ENV PATH "${rustup_dir}/bin:${PATH}"
```

Comment:
In general, it is best practice to add items to the end of the `PATH` rather than the beginning. The reason for this is that the OS searches for binaries in `PATH` order and usually places more common locations at the beginning of the `PATH`. Only rust tools will be found in `${rustup_dir}/bin`, so it will slow down lookup of most tools to place it first.

Reply:
done.